.github/workflows/test-c3.yml

```diff
@@ -0,0 +1,40 @@
+name: C3 Tests
+
+on:
+  pull_request:
+    paths:
+      - packages/create-cloudflare
+
+env:
+  node-version: latest
+
+jobs:
+  check:
+    name: "Checks"
+    runs-on: ubuntu-latest
```

Comment:
Perhaps run it on Windows too?

Reply:
Good suggestion. I copied over the matrix strategy from the PR workflow so it'll do ubuntu, windows, and mac. Haven't had a change to test whether the tests actually run on Windows though 😬